pkg/infra/Makefile

```diff
@@ -0,0 +1,15 @@
+BINARY_NAME=main
+ 
+build:
+		go mod tidy
+		go build -o ${BINARY_NAME} main.go
+
+run:
+	  go run main.go
+
+runWithJSON:
+		go run main.go -j
+ 
+clean:
+		go clean
+		# rm -f ${BINARY_NAME}
```

Comment:
Structure wise (just to keep it as clean as possible), should we put the Makefile into the `\scripts` folder and the executable outputs into the `\cmd` directory?

Reply:
This comment has some merit but this PR has already been merged. Can you create a new PR or Issue for this point please?